include/mfmg/sparse_matrix_device.cuh

```diff
@@ -48,6 +48,9 @@ public:
 
   ~SparseMatrixDevice();
 
+  SparseMatrixDevice<ScalarType> &
+  operator=(SparseMatrixDevice<ScalarType> &&other);
```

Comment:
Why are you providing a move assignment operator here?  You might want to do that in a separate commit.

Reply:
Because in `mmult` I need to do `C = convert_matrix()` and I don't want to copy the matrix.